sae.c

```diff
@@ -76,6 +76,13 @@
 #define CN_Update(ctx,x,l) HMAC_Update((ctx),(x),(l))
 #define CN_Final(ctx,x) HMAC_Final((ctx), (x), &function_mdlen)
 
+/* Pre 1.1.0 compatibility macros */
+#ifndef OPENSSL_API_COMPAT
```

Comment:
where does this get set?

Reply:
This is introduced in 1.1.0, defined `include/openssl/opensslconf.h`, and included in this module by `openssl/hmac.h`